src/api/domain/withdrawals-api.ts

```diff
@@ -66,11 +66,11 @@ export const WithdrawalsApiAxiosParamCreator = function (configuration?: Configu
             const localVarHeaderParameter = {} as any;
             const localVarQueryParameter = {} as any;
 
-            if (xImxEthAddress != null) {
+            if (xImxEthAddress !== undefined && xImxEthAddress !== null) {
```

Comment:
What's all this? Is this necessary? Worried it might get blown away after future codegen.

Reply:
I believe this is outdated now https://github.com/immutable/imx-core-sdk/blob/coresdk-interface-class/src/api/domain/withdrawals-api.ts#L69